test/WebJobs.Script.Tests.Integration/TestScripts/Node/ManualTriggerStripsFunctions/function.json

```diff
@@ -0,0 +1,15 @@
+﻿{
+    "bindings": [
```

Comment:
Can you just stuff this test function in the Scenarios function like we do for all of these one off scenarios? :)